swiftgen-cli/storyboards.swift

```diff
@@ -15,19 +15,21 @@ let storyboardsCommand = command(
     description: "The name of the enum to generate for Scenes"),
   Option<String>("segueEnumName", "StoryboardSegue", flag: "g",
     description: "The name of the enum to generate for Segues"),
-  Options<String>("import", [], count: 0,
+  VariadicOption<String>("import", [],
```

Comment:
🤔 I guess that means that #175 wasn't actually working as described in the CHANGELOG entry until now… (« Multiple imports can be added by repeating this flag. ») and that actually fixes it then?